.release_metadata.json

```diff
@@ -1,3 +1,3 @@
 {
-    "timestamp": "2025-01-16 13:55:53"
+    "timestamp": "2025-01-27 14:59:20"
```

Comment:
Should we put a timezone to avoid ambiguity?

Reply:
Added